pydantic/fields.py

```diff
@@ -279,6 +299,8 @@ def __repr_args__(self) -> ReprArgs:
                 continue
             elif s == 'repr' and self.repr is True:
                 continue
+            elif s == 'final':
```

Comment:
This causes it to not show `final` in the repr for `FieldInfo`. I can undo this before merging if desired, it just requires some tests of repr to be updated which I figured wasn't helpful at this stage.

Reply:
Up to you, is there a particular reason to omit "final"?

Reply:
I think maybe we should remove it anyway, as noted above.